interface/client/styles/layout.import.less

```diff
@@ -118,16 +105,6 @@ main {
     left: @widthSideBar;
     top: 0; 
     bottom: 0;
-
-    &.open-bar {
```

Comment:
back in time when you focused on the url bar, a huge div unfolded, and blurred the active tab. as we don't have this unfolding section anymore, i did the clearance.